specs/vep-2272-complete-data-job-configuration-persistence/README.md

```diff
@@ -136,6 +139,7 @@ replicated:
 * `deployed_by`
 * `deployed_date`
 * `deployment_version_sha`
+* `mode`
```

Comment:
This property isn't retained within Kubernetes, nor is it utilized anywhere within the system, leading me to lean towards discarding it.

Reply:
It's used in supercollider , currently parsed out of config.ini. It is not stored in the database, yet.


But it would be soon  around the config changes we are planning in https://github.com/vmware/versatile-data-kit/issues/2420 if not in this initiative.

Reply:
It is only passed to the API but not used for anything right?

Reply:
It's used in supercollider-vdk.